docs/spfx/use-theme-colors-in-your-customizations.md

```diff
@@ -103,7 +83,71 @@ When you scaffold a new SharePoint Framework client-side web part, by default, i
 
     ![Side-by-side view of the same web part displayed in two sites using different colors. The web part follows the color scheme of each website](../images/themed-styles-side-by-side.png)
 
+### Support section backgrounds
+
+A customziation utilizing theme colors may still stand out if it doesn't respond to section background changes via theme variants support. See [Supporting Section Backgrounds](./webparts/guidance/supporting-section-backgrounds.md) for more information.
+
+A web part without theme variants support uses a white background regardless of the selected section background color.
+
+![customization without theme variant support](../images/customization-without-theme-variants-support.png)
+
+A web part configured to support theme variants can apply the section background to the web part background.
+
+![customization with theme variant support](../images/customization-with-theme-variants-support.png)
+
+## Available theme tokens and their occurances
+
+In the context of modern SharePoint sites, the values associated with theme tokens have numerous occurances within front-end, system page, and SharePoint web part UI. To best match your customizations to the general look and feel, consideration should be made to using appropriate theme tokens.
+
+The following theme tokens are available for you to use:
+
+_Note: default values below correspond to the defaults on a modern team site when using the **red palette**._

```

Comment:
Please use the MSDocs alert format for notes. It should look like this (including the blockquote). Also, please use complete sentences in the note. For example, this should be:

```
> [!NOTE]
> The default values below correspond to the defaults on a modern team site when using the **red palette**.
```